src/Features/CSharp/Portable/SignatureHelp/AbstractOrdinaryMethodSignatureHelpProvider.cs

```diff
@@ -79,7 +79,7 @@ internal abstract class AbstractOrdinaryMethodSignatureHelpProvider : AbstractCS
                 result.Add(Space());
             }
 
-            result.AddRange(method.ToMinimalDisplayParts(semanticModel, position, MinimallyQualifiedWithoutParametersFormat));
+            result.AddRange(method.ToMinimalDisplayParts(semanticModel, position, MinimallyQualifiedWithoutParametersFormat.AddMiscellaneousOptions(SymbolDisplayMiscellaneousOptions.AllowDefaultLiteral)));
```

Comment:
this is the 'without parameters' portion.  i.e. it affects the part of the signature up *to* the `(`.  so it wouldn't/shouldn't affect parameters.

Reply:
@CyrusNajmabadi Oh. Thanks for pointing this out.

Probably I should start debugging in https://github.com/dotnet/roslyn/blob/4d24556df984774dc70cf707eff1ee9d3d96b9b9/src/Features/CSharp/Portable/SignatureHelp/InvocationExpressionSignatureHelpProvider.cs

Reply:
@CyrusNajmabadi I added a test in InvocationExpressionSignatureHelpProviderTests:

```
        [Fact, Trait(Traits.Feature, Traits.Features.SignatureHelp)]
        public async Task TestInvocationOnTriggerParens_OptionalDefaultStruct()
        {
            var markup = @"
class Program
{
   static void SomeMethod(CancellationToken token = default) => throw new NotImplementedException();

    static void Main(string[] args)
    {
        [|SomeMethod($$|]);
    }
}";

            var expectedOrderedItems = new List<SignatureHelpTestItem>();
            expectedOrderedItems.Add(new SignatureHelpTestItem("void Program.SomeMethod([CancellationToken token = default])", string.Empty, null, currentParameterIndex: 0));

            await TestAsync(markup, expectedOrderedItems, usePreviousCharAsTrigger: true);
        }
```

It fails with the following:

```
  Error Message:
   Assert.Equal() Failure
                                  (pos 51)
Expected: ···lationToken token = default])
Actual:   ···lationToken token = null])
                                  (pos 51)
```

The actual result from the test doesn't match the actual result that actually happens, and also doesn't match the expected behavior. Am I testing the wrong thing? 😄 

In any case, it's weird to see `CancellationToken token = null` as it's a non-nullable value type.

Reply:
That's a funny mistake :smile:
I'll fix that and continue trying to debug this locally.
Thanks a lot for pointing to my mistake.